z_model.php

```diff
@@ -136,14 +136,14 @@ function getResult() {
          * @return int Number of results in the last query
          */
         function countResults() {
-            return $this->z_db->countResults(...func_get_args());
```

Comment:
Not a docs change

Reply:
`countResults` does not take any arguments, which I have noticed due to the docs change, `func_get_args` should not be used there as there is nothing to pass.